iina/MPVOption.swift

```diff
@@ -6,462 +6,102 @@ struct MPVOption {
     static let alang = "alang"
     /** --slang=<languagecode[ */
     static let slang = "slang"
-    /** --vlang=<...> */
-    static let vlang = "vlang"
     /** --aid=<ID|auto|no> */
```

Comment:
Generated source, must not be modified.

Reply:
Fixed in ce4a8bca1c16f0f7077a75b0fbe75b010cc44c95